mne/realtime/client.py

```diff
@@ -1,304 +1,65 @@
-# Authors: Christoph Dinh <chdinh@nmr.mgh.harvard.edu>
-#          Martin Luessi <mluessi@nmr.mgh.harvard.edu>
-#          Matti Hamalainen <msh@nmr.mgh.harvard.edu>
+# Authors: Teon Brooks <teon.brooks@gmail.com>
+#          Mainak Jas <mainak@neuro.hut.fi>
```

Comment:
I think the diff is really unreadable if we rename the files. I'd prefer keeping the filenames as is for now ...